frontend/src/types.ts

```diff
@@ -145,3 +145,42 @@ export interface updateBookingHoursBody {
   startWeek: number;
   endWeek?: number;
 }
+
+export interface ProjectWithConsultantsReadModel {
+  projectName?: string | null;
+  customerName?: string | null;
+  bookingType?: ProjectState;
+  consultants?: ConsultantReadModel[] | null;
+  isBillable?: boolean;
+}
+
+export interface EngagementWriteModel {
+  consultantIds?: number[] | null;
+  bookingType?: ProjectState;
+  isBillable?: boolean;
+  projectName?: string | null;
+  customerName?: string | null;
+}
+
+export enum ProjectState {
+  Closed = "Closed",
+  Order = "Order",
+  Lost = "Lost",
+  Offer = "Offer",
+  Active = "Active",
+}
+
+export interface ConsultantReadModel {
```

Comment:
Hva er forskjellen på denne og Consultant? 😄

Reply:
Hmm. Godt poeng 🤔 Har bare slavisk copy/pasta det som swagger-generate ga meg. 
Liker vi at typene heter det samme front/backend for øvrig? @jonasbjoralt @sigtheidiot

Reply:
Tenker for Consultant og Organisation er det rart i Frontend om de heter ConsultantReadModel/OrganisationReadModel, men skjønner om det blir lettere å jobbe med hvis de heter det samme 😅 (Grunnen til at jeg la merke til det var vel fordi jeg hadde min egen "ConsultantMultipleCellReadModel" som jeg oppdaget var lik Consultant 😂 )

Reply:
Kan godt endre det, men hadde foretrukket at vi har en "policy" på hvordan vi håndterer typing her, så vi slipper å lage så mye customs.

Reply:
Njæ.. tok å fjerna det :) Ser swagger genererer lite "required" fields. Nesten alt er så optional.. 🤔

Reply:
ReadModel er en "modell" eller "projeksjon" av Consultant, men ikke det samme som objektet vi har i koden. For eksempel, er `department` en tekst her, a la "Trondheim", mens det er et objekt av klassen `Department` i Consultant. 

`bookings` og `detailedBookings` finnes heller ikke i Consultant, siden disse er spisset mot hva vi har behov for å vise frontend